app/resources/[slug]/page.tsx

```
@@ -263,36 +341,36 @@ export default async function ArticlePage({ params }: { params: { slug: string }
-              prose-pre:bg-gray-50 dark:prose-pre:bg-gray-700/50 prose-pre:border prose-pre:border-gray-200 dark:prose-pre:border-gray-600 
+              prose-pre:bg-gray-50 dark:prose-pre:bg-gray-700/50 prose-pre:border prose-pre:border-gray-200 dark:prose-pre:border-gray-600
               prose-pre:rounded-xl prose-pre:shadow-sm
               prose-table:border-collapse prose-table:overflow-hidden prose-table:w-full
               prose-th:bg-gray-50 dark:prose-th:bg-gray-800 prose-th:p-3 prose-th:text-left
               prose-td:p-3 prose-td:border-b prose-td:border-gray-200 dark:prose-td:border-gray-700
               prose-tr:hover:bg-gray-50 dark:prose-tr:hover:bg-gray-800/50
               prose-hr:my-8">
-              <div 
-                dangerouslySetInnerHTML={{ __html: article.contentHtml }} 
+              <div
+                dangerouslySetInnerHTML={{ __html: article.contentHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)